### ISSUE TITLE
Explorer ERC-1155 Token List Not Visible 

### ISSUE THREAD

**Issue by MarcusWentz** [Shardeum/shardeum-bug-reporting#14]
##### Overview

Unable to click on ERC-1155 token transactions for an address after tokens are added. 
The text for this option is colliding with another token option, making it hard to read and see.

![erc1155](https://user-images.githubusercontent.com/52706599/210121454-72e64aee-92c4-44cc-a516-b754ee0df6fe.png)

##### Context 

Deployed the following Solidity contract from Remix IDE:

```solidity
// SPDX-License-Identifier: MIT
pragma solidity 0.8.17;
import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";

error invalidFruitIdNumber();

contract FruitStore is ERC1155 { //ERC1155 cannot also be a ERC1155Holder for having tokens ERC1155 tokens sent to it.

    uint public constant Raspberry = 0; // "constant" and "immutable" variables don't use storage slots, which saves gas.
    uint public constant Watermelon = 1; // "constant" variables can never change.
    uint public constant Blueberry = 2;
    uint public constant Avacado = 3;
    uint public constant Guava = 4;
    uint public constant Pitaya = 5;
    uint public constant Apple = 6;
    uint public constant Banana = 7;
    uint public immutable ExtraFruitId; //"immutable" can only be changed in the constructor.

    constructor(uint extraFruitIdPick) ERC1155("https://en.wikipedia.org/wiki/{name}") { //When using safeTransferFrom, have data as 0x00.
        if(extraFruitIdPick > 7) { revert invalidFruitIdNumber(); } //Don't check for negative since uint must be 0 or positive.
        ExtraFruitId = extraFruitIdPick; //"immutable" can only be changed in the constructor.
        _mint(msg.sender, Raspberry,    100*(1 ether), "");  //To send tokens from wallet to wallet, send the following Tx to this contract: "setApprovalForAll(walletToSendTo,true)".
        _mint(msg.sender, Watermelon,   100*(1 ether), "");  //Then you can transfer tokens with: "safeTransferFrom(fromAddress,toAddress,id,amount,0x00)".
        _mint(msg.sender, Blueberry,    100*(1 ether), "");  //Smart contracts must inherit "ERC1155Holder.sol" to receive ERC1155 tokens.
        _mint(msg.sender, Avacado,      100*(1 ether), "");
        _mint(msg.sender, Guava,        100*(1 ether), "");
        _mint(msg.sender, Pitaya,       100*(1 ether), "");
        _mint(msg.sender, Apple,        100*(1 ether), "");
        _mint(msg.sender, Banana,       100*(1 ether), "");
        _mint(msg.sender, ExtraFruitId, 100*(1 ether), "");
    }

}
``` 

##### Resources

Deployed ERC-1155 token transaction: https://explorer-liberty20.shardeum.org/tx/0xc7ff842dcb8a39be625208169dd6cf151be64cba437b4d5a9695a4ab0ee482fd

**Comment by MarcusWentz** [comment]
ERC-1155 tokens do not appear in Betanet 1.X Explorer either after a 

```solidity
safeBatchTransferFrom(address from, address to, uint256[] ids, uint256[] amounts, bytes data)
``` 

transaction:

https://explorer-sphinx.shardeum.org/transaction/0x56eb980bc57c0813dc99c9e58e37a6b606bdbdb447ec749d70d59974d81079a5

**Comment by MarcusWentz** [comment]
Test with multiple tokens transferred to a single "to" address (batch can not bulk send with a "to" array):

ERC-1155 tokens do not appear in Betanet 1.X Explorer either after a:

```solidity
safeBatchTransferFrom(
                      address from, 
                      address to, 
                      uint256[] ids, 
                      uint256[] amounts, 
                      bytes data
)
```

with inputs:

```solidity
safeBatchTransferFrom(
                      0xc1202e7d42655F23097476f6D48006fE56d38d4f,
                      0x66C1d8A5ee726b545576A75380391835F8AAA43c,
                      [5,6,7], 
                      [1,2,3], 
                      0x
)
```

transaction:

https://explorer-sphinx.shardeum.org/transaction/0x4af21643949f0403fc84e6f6a12ff21f60e06349ed8f856e2690d929ca01a3e3

**Comment by swarna1101** [comment]
![image](https://github.com/Shardeum/shardeum-bug-reporting/assets/113277972/1954ec6d-ae85-48ff-8426-00ca0c0e4cfd)

Its solved in network version 1.7.1 you can check it
